spec/support/helpers/authentication_helper.rb

```diff
@@ -0,0 +1,8 @@
+module AuthenticationHelper
+  def sign_in_as(user)
```

Comment:
Add `visit "/"` to the beginning of this method as well. (I know it was not on the example that I showed you, but that was 9 months ago D-Rod begin stupid)